creator-node/src/serviceRegistry.js

```diff
@@ -71,6 +71,9 @@ class ServiceRegistry {
     // init libs
     this.libs = await this._initAudiusLibs()
 
+    // Transcode handoff requires libs. Set libs in AsyncProcessingQueue after libs init is complete
+    this.asyncProcessingQueue.setLibs(this.libs)
```

Comment:
what if we get libs from the service registry?

Reply:
I think if we do it your suggestion, that the AsyncProcessingQueue fn calls will load as undefined throughout the code base though :/

Reply:
what do you mean by this? nothing should get added to this queue until after libs is initialized because this would run in serviceregistry before the web server comes up